Fix made to MKE CLI install using script code-block. (#77)
Mirantis/mke-docs: content/docs/getting-started/install-MKE-CLI.md

weight: 2
---

Before you can proceed with the MKE installation, you must download and install

> `mkectl`, the MKE CLI tool, as well as `kubectl` and `k0sctl`. You can do this
> automatically, using the script provided in the section that immediately
> follows, or you can do it manually.

Reviewer: ```suggestion
`mkectl`, the MKE CLI tool, as well as `kubectl` and `k0sctl`. You can do this
[automatically](#install-using-script), using the provided script, or you can do it [manually](#install-manually) .
```

> ## Install using script

Reviewer: Should'n it be titled “Install automatically” if the next one is “Install manually”?

> To automatically install the necessary dependencies, you can use the
> `install.sh` script offered in the example procedure that follows.

Reviewer: ```suggestion
Install the necessary dependencies automatically, you can use the
`install.sh` script shown in the example:
```

Author: Tweaked to read as follows:

"To automatically install the necessary dependencies, you can use an `install.sh` script, as exemplified in the following procedure:"

The reason being that the procedure is not actually an _example_.

Download `mkectl` from the S3 bucket:

| Distribution | Architecture | Download |
|--------------|--------------|----------|
| Linux        | arm64        | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_linux_arm64.tar.gz) |
| Linux        | x86_64       | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_linux_x86_64.tar.gz) |
| MacOS        | arm64        | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_darwin_arm64.tar.gz) |
| MacOS        | x86_64       | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_darwin_x86_64.tar.gz) |
| Windows      | arm64        | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_windows_arm64.zip) |
| Windows      | x86_64       | [download](https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/v4.0.0-alpha.4.0/mkectl_windows_x86_64.zip) |

Envisioned as a single binary that is capable of managing MKE clusters without
any additional dependencies, the MKE CLI also requires that you have the

> following tools installed on your system:

Reviewer: Could you rephrase the fragment to read more technical, currently it is a bit too commercial Maybe:
MKE CLI is a single binary that is capable of managing MKE clusters without
any additional dependencies. To use it, you need to install the following tools on your system:

Author: Good thought. I tweaked the text to read:

"The MKE CLI is a single binary that is capable of managing MKE clusters without any additional dependencies. Its use, though, requires the presence of the following tools on your system:"
